tests/data/dsl-syntax-validation-cases.yaml

```diff
@@ -419,24 +419,6 @@
         end: 17
       metadata:
         symbol: "reader"
-- name: unsupported schema version should yield error
```

Comment:
This now becomes a validation error, however we can't really keep it because we don't have validation in the Go side of things

Reply:
Can you move it to: https://github.com/openfga/language/blob/main/tests/data/dsl-semantic-validation-cases.yaml